src/routes/HomePage.tsx

```diff
@@ -0,0 +1,3 @@
+export const HomePage = () => (
+  <h1 className="title">Home page</h1>
+);
```

Comment:
Let's use our container here

```suggestion
export const HomePage = () => (
  <div className="container">
    <h1 className="title">Home page</h1>
  </div>
);
```